src/components/Patient/PatientRegistration.tsx

```
@@ -0,0 +1,1048 @@
+              onValueChange={(value) =>
+                setForm((f) => ({ ...f, gender: Number(value) }))
+              }
+              errors={errors["gender"]}
+              className="flex items-center gap-4"
+            >
+              {GENDER_TYPES.map((g) => (
+                <>
+                  <RadioGroupItem
+                    value={g.id.toString()}
+                    id={"gender_" + g.id}
+                  />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)